src/background.js

```diff
@@ -289,6 +289,16 @@ function getRootDomain(url) {
 
 }
 
+function getFrameAncestorsURLAndCheckIfFacebook(frameAncestorsArray) {
+  if (!frameAncestorsArray || frameAncestorsArray.length === 0) {
+    // No frame ancestor return false
+    return false;
+  }
+
+  const frameAncestorsURL = frameAncestorsArray[0].url;
+  return isFacebookURL(frameAncestorsURL);
```

Comment:
We could remove this variable. It's how we're doing it on L518. 

```suggestion
  return isFacebookURL(frameAncestorsArray[0].url);
```

Reply:
Might be a slight performance optimization. Is this called on every single request?

Reply:
Yes – In `blockFacebookSubResources()`, the `isFacebookURL()` function is called three times on every request: 
- `requestDetails.originUrl`
- `requestDetails.url`
- and this one. 

Note that there are [some short-circuits](https://github.com/mozilla/contain-facebook/blob/master/src/background.js#L498-L540) in these checks, so sometimes it doesn't get far into the function before bailing.